config/importmap.rb

```diff
@@ -5,3 +5,7 @@
 
 pin 'datatables'
 pin 'stupidtable'
+
+pin '@hotwired/stimulus', to: 'stimulus.min.js'
+pin '@hotwired/stimulus-loading', to: 'stimulus-loading.js'
```

Comment:
as these are standard rails importa, I'd move them to the top w/ https://github.com/hotwired